### DISCUSSION TITLE
Total Fuel Burn Issue

### DISCUSSION THREAD

**Discussion by jetstarpilotmel** [flybywiresim/aircraft#5477] [Help] [answered]
Hello,

The total fuel burn located on the fuel page does not reset once the engines have been turned on again (when completing a second flight with the fbw a320 aircraft)

In real life the total fuel burn for each engine and the total fuel burn for both engines resets once you turn the engines on.

I am running the latest development version with WU5. Note that this issue was occurring before WU5 and still exists. 

Cheers
James

**Comment by tracernz** [comment, marked as answer]
It would be best to open a GitHub bug report for this.
